scripts/lobby.gd

```diff
@@ -16,16 +16,29 @@ func _ready():
 	assert(DSGNetwork.connect_websocket("ws://%s/lobby/%s/join" % [ws_address, GlobalVariables.id]))
 
 func _on_start_game_button_pressed():
-	var _start_message = {
-		"type": "start_game",
-		"payload": {}
+	var start_message = {
+		"type": "host_start_game",
+		"payload": {
+			"platform": {
+				"tiles": [
+					{
+						"position": {
+							"x": 0,
+							"y": 0
+						},
+						"texture_id": "grass",
+						"tile_type": "void"	
+					}
+				]
+			}
+		}
 	}
-	# TODO: type `start_game` is not implemented yet on the server side
-	# DSGNetwork.send(JSON.stringify(start_message).to_utf8_buffer())
+	DSGNetwork.send(JSON.stringify(start_message).to_utf8_buffer())
```

Comment:
why doesn't `DSGNetwork.send` take a `Variant` or perhaps even a `Dictionary`?

Reply:
good point, I've changed that in bd26406